src/util/partial_store.h

```diff
@@ -36,6 +36,9 @@
 
 static really_inline
 void partial_store_u32(void *ptr, u32 value, u32 numBytes) {
+    if(ptr == NULL){
```

Comment:
These are functions that are called in critical loops, better add an assert and a NOLINT.